src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ParameterConverter.cs

```diff
@@ -81,6 +81,22 @@ internal static class ParameterConverter
             return literal;
         }
 
+        internal static string? GetDefault(string? dataType)
+        {
+            return dataType switch
+            {
+                string s when string.IsNullOrEmpty(s) => null,
+                string s when s.Equals("bool", StringComparison.OrdinalIgnoreCase) => false.ToString(),
+                string s when s.Equals("choice", StringComparison.OrdinalIgnoreCase) => string.Empty,
```

Comment:
shouldn't it be `null` for choice?

Reply:
`null` for choice means that default value should be used.
If defualt would be `null` as well - it would be recognized as error: https://github.com/dotnet/templating/blob/main/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ParameterConverter.cs#L147-L148

`string.Empty` is explicit unset - sounds as reasonable default